views/teams/topics.hbs

```diff
@@ -29,17 +29,17 @@
         <div class="col-sm-6">
             {{#userHasPermission "COURSE_EDIT"}}
             <a href="/teams/{{../_id}}/topics/add" class="btn btn-secondary btn-sm pull-right ml-1"
-                title="Thema hinzufügen">
+                title="{{$t "teams.topics.button.addTopic"}}">
                 <i class="fa fa-plus" aria-hidden="true"></i>
-                <span class="hidden-lg-down">Thema hinzufügen</span>
+                <span class="hidden-lg-down">{{$t "teams.topics.button.addTopic"}}</span>
             </a>
-            <button data-courseId="{{../_id}}" class="btn btn-secondary btn-sm pull-right ml-1 btn-import-topic"
```

Comment:
why this was changed to a?

Reply:
All changes happened because of merging. I havent done this intentionally! should have checked all the changes carefully.